src/Microsoft.SqlTools.ServiceLayer/Localization/sr.resx

```diff
@@ -1768,4 +1768,25 @@
     <value>Could not find the schema compare session to cancel</value>
     <comment></comment>
   </data>
+  <data name="SqlAssessmentOperationExecuteCalledTwice" xml:space="preserve">
+    <value>A SQL Assessment operation&apos;s Execute method should not be called more than once</value>
+    <comment></comment>
+  </data>
+  <data name="SqlAssessmentGenerateScriptTaskName" xml:space="preserve">
+    <value>Generate SQL Assessment script</value>
+    <comment></comment>
+  </data>
+  <data name="SqlAssessmentQueryInvalidOwnerUri" xml:space="preserve">
+    <value>Not connected to a server</value>
+    <comment></comment>
+  </data>
+  <data name="SqlAssessmentConnectingError" xml:space="preserve">
+    <value>Cannot connect to the server</value>
+    <comment></comment>
+  </data>
+  <data name="SqlAssessmentUnsuppoertedEdition" xml:space="preserve">
+    <value>Unsupported engine edition {0}</value>
+    <comment>.
+ Parameters: 0 - editionCode (int) </comment>

```

Comment:
Did you add this comment yourself? That period seems wrong - if you did could you remove it and just have the comment on one line?

Reply:
No, it was generated. We can see it on many other lines with parameterized strings.  [1626](https://github.com/microsoft/sqltoolsservice/blob/cf6eec47d3a195f906706668ae1ee327494c2936/src/Microsoft.SqlTools.ServiceLayer/Localization/sr.resx#L1626)

Reply:
master branch contains the dot as well. [sr.resx#L320](https://github.com/microsoft/sqltoolsservice/blob/0a53ff9ff7241156a88c1311e352042480bdc1db/src/Microsoft.SqlTools.ServiceLayer/Localization/sr.resx#L320)